components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3195,6 +3195,10 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         throw new ResponseError(ErrorCodes.SAAS_FEATURE, `Not implemented in this version`);
     }
 
+    async setUsageAttribution(ctx: TraceContext, usageAttributionId: string): Promise<void> {
+        throw new ResponseError(ErrorCodes.SAAS_FEATURE, `Not implemented in this version`);
```

Comment:
Why is this not implemented in non SaaS?

Reply:
right, that's mistake. it should be the default.

Reply:
check